src/pages/Register/index.jsx

```diff
@@ -1,7 +1,194 @@
-import React from 'react';
+import React, { useState } from "react";
+import { Form, Input, Button, Radio, Row, Col } from "antd";
+import { useDispatch } from "react-redux";
+import { Link, Navigate, useNavigate } from "react-router-dom";
+import { LockOutlined } from "@ant-design/icons";
 
-function Register() {
-  return <p>Register</p>;
-}
+import { registerAction } from "../../redux/actions";
+
+import "../Login/style.scss";
+import "./style.scss";
+
+const Register = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const handleRegister = (values) => {
+    dispatch(
+      registerAction({
+        data: {
+          name: values.name,
+          surname: values.surname,
+          phone: values.phone,
+          email: values.email,
+          password: values.password,
+        },
```

Comment:
```suggestion
  const handleRegister = (data) => {
    const { confirm, ...rest } = data
    dispatch(
      registerAction({
        data: rest,
```
nếu muốn lấy hết tất cả, chỉ loại bỏ mình confirm password thì làm như này cho gọn nhé